src/define-colors.lisp

```diff
@@ -0,0 +1,17 @@
+
+(in-package :cl-ansi-text)
+
+(defmacro define-colors ()
```

Comment:
why a separate file?

Reply:
I am not a big fan of `eval-when` used just for making the constants evaluated in the compile time. Putting this code in a separate file makes the dependency explicit